plugins/modules/zos_operator.py

```diff
@@ -35,10 +35,28 @@
     default: false
   debug:
     description:
-      - Return debugging information.
+      - Return rexx debugging information.
     type: bool
     required: false
     default: false
+  security:
+    description:
+      - Call command with (verbose) operator which returns security information.
+    type: bool
+    required: false
+    default: false
+  delay:
+    description:
+      - Set maximum time in seconds to wait for response.
```

Comment:
I misread that... I'll add '(WAIT)' to the call line, which should do that.  I wouldn't have noticed, because I was sending singular commands.  Thanks!

Reply:
@richp405 - I see you added the WAIT if verbose is selected unbeknown the the user; In my mind I had assumed we would expose this is as a TRUE/FALSE with default TRUE given the impact it can have if someone sets a very long DELAY. Open to discussion on that, but we might be hindering performance with always setting WAIT without out opportunity to override. 

```
WAIT
    specifies that SDSF should wait the full delay interval before retrieving responses. This option is strongly recommended to ensure the responses are accessible in the ISFULOG special variable. The delay interval is specified with the ISFDELAY variable.
```

Reply:
Thank you, see WAIT added, I know it can seem a bit odd we ask for a delay then basically a max we will wait on the delay but rather than forcing user to wait the full time for an overly large delay , they may rather return and that is where i see value. As they get familiar with their system and feeling out how long it will take plus a few extra seconds they may want to return sooner than later.